packages/react-storage/package.json

```diff
@@ -58,7 +58,7 @@
       "name": "StorageManager",
       "path": "dist/esm/index.mjs",
       "import": "{ StorageManager }",
-      "limit": "25 kB"
+      "limit": "26 kB"
```

Comment:
Can we be tighter with this limit? I htink it only wnet over by .05kB, so maybe 25.2?